Gopkg.lock

```diff
@@ -1,194 +0,0 @@
-# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
-
```

Comment:
Any significant dependency version changes in the switch or not really? hard to tell from the diff